scripts/west/west-commands.yml

```diff
@@ -10,3 +10,8 @@ west-commands:
       - name: zap-gui
         class: ZapGui
         help: Run Matter ZCL Advanced Platform (ZAP) GUI
+  - file: scripts/west/zcl_generate.py
+    commands:
+      - name: zcl-generate
+        class: ZCLGenerate
+        help: Generate ZCL code from XML cluster definition file
```

Comment:
I think naming is a bit unfortunate here and there. According to name and help it seems that it generates source code, while it's not true, as it generates only json file that describes data model. Source code is generated by zap-generate, which is also unfortunate name, because it sounds like it could generate zap file, not the code.

I don't know if we should change zap-generate name, as customers could already learn how to use it, but I would try to name the new command in a different way (and change help as well). Maybe zap-generate-json?

Reply:
Or maybe `zcl-append-cluster`

Reply:
But other commands start with zap, so why we want to use zcl now?

Reply:
Because, actually it is more about ZCL than zap.

Reply:
Changed to zap-append :)